config_files/ps-extractor.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
 
 folder=$1
-destination="/var/www/html"
 
-if [[ -n "$folder" ]]; then
-    if [ -d $folder/prestashop ]; then
-        cp -n -R $folder/prestashop/* $destination
-    else
-        unzip -n -q $folder/prestashop.zip -d $destination
+if [[ -n "$folder" ]]; then  
+
+    # dwl version contains zip file with tree structure (1.7)
+    if [ ! -d $folder/prestashop ]; then
+        unzip -n -q $folder/prestashop.zip -d $folder/prestashop
+	rm -rf $folder/prestashop.zip	
     fi
+
+    # prepair tree structure for volumes
+    mv $folder/prestashop/themes $folder/prestashop/modules $folder/prestashop/override $folder/
+
+    # build relative symlinks for volumes
+    ln -s ../themes $folder/prestashop/themes
+    ln -s ../modules $folder/prestashop/modules
+    ln -s ../override $folder/prestashop/override
```

Comment:
Some modules won't work because of those symlink. If we can avoid symlinks it's better

The module that won't work will need to be updated to remove all code that include files using relative path like that:
dirname(__FILE__).'/../../config/config.inc.php

Because the symlink is resolved before following the relative path.
We expect the file to be loaded from:
prestashop/config/config.inc.php
but it will look for it in:
config/config.inc.php